src/redisraft.c

```diff
@@ -659,6 +724,15 @@ static void handleRedisCommandAppend(RedisRaftCtx *rr,
         return;
     }
 
+    bool deny_oom = getDenyOomStatus(rr, ctx, cmds);
```

Comment:
Please take a look at https://github.com/RedisLabs/redisraft/blob/1ef16de3c847d4ece2dccfdcbc43da8254f726c1/src/commands.c#L196

We can define `CMD_SPEC_DENY_OOM` and add it to command flags. Otherwise, we cannot afford fetching "deny-oom" on append time.

Reply:
If we are doing it anyway, so yes - it will be easy to maintain such a dictionary, but I wonder what will be the result for module commands - it probably depends on the order we load them, did we have a solution for that? (it's relevant also for `populateCommandSpecFromRedis`) @tezc

Reply:
k, anyway I suggest taking out the `populateCommandSpecFromRedis` call from the config path.

Reply:
restuctured to run just "commands" and cache everything at startup and at module load time